test/unit/BMath.t.sol

```diff
@@ -4,564 +4,263 @@ pragma solidity 0.8.25;
 import {BMath, BNum} from 'contracts/BMath.sol';
 import {Test} from 'forge-std/Test.sol';
 
-// For test contract: expose the internal functions of BNum
-contract BNumExposed is BNum {
-  function bdivExposed(uint256 a, uint256 b) external pure returns (uint256) {
-    return super.bdiv(a, b);
-  }
-
-  function bmulExposed(uint256 a, uint256 b) external pure returns (uint256) {
-    return super.bmul(a, b);
-  }
-
-  function bsubExposed(uint256 a, uint256 b) external pure returns (uint256) {
-    return super.bsub(a, b);
-  }
-
-  function baddExposed(uint256 a, uint256 b) external pure returns (uint256) {
-    return super.badd(a, b);
-  }
-}
-
 // Main test contract
 contract BMathTest is Test {
   BMath bMath;
-  BNumExposed bNum;
 
-  uint256 BONE; // The fixed-point unit for BMath and BNum
+  uint256 constant BONE = 1e18; // The fixed-point unit for BMath and BNum
+
+  // valid scenario
+  uint256 constant weightIn = BONE;
```

Comment:
rm linter rule?

Reply:
go for it! would you disable it _entirely_ or only for constants? I'd prefer the latter